### PR TITLE
[TD-13928]<feature>improve user experience for taosbenchmark

### PR DESCRIPTION
**Pull request by zhaoyanggh**

### CODE REVIEW

File: tests/develop-test/5-taos-tools/taosbenchmark/json/stmt_insert_alltypes.json [hunk 1]
```diff
@@ -43,8 +43,8 @@
       "data_source": "rand",
       "insert_mode": "stmt",
       "line_protocol": "line",
-      "childtable_limit": 0,
-      "childtable_offset": 0,
+      "childtable_limit": -10,
```
Comment (sangshuduo) [inline tests/develop-test/5-taos-tools/taosbenchmark/json/stmt_insert_alltypes.json]: ?

**Review by sangshuduo — CHANGES_REQUESTED** [review]
see my comment

**Comment by zhaoyanggh** [comment]
> see my comment

fixed

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**